components/datetime/tests/fixtures/tests/styles.json

```diff
@@ -95,13 +65,13 @@
             "value": "2020-03-21T08:25:07.000",
             "options": {
                 "style": {
-                    "time": "full",
-                    "date": "full"
+                    "time": "short",
+                    "date": "medium"
```

Comment:
`DateTimeFormat` no longer supports patterns with time zones, so it cannot use `Time::Long` or `Time::Full`.

These are tested by `ZonedDateTimeFormat` which pulls from `styles_with_zones.json`.